### ISSUE TITLE
ZFS pool is not shown in Disks section

### ISSUE THREAD

**Issue by wezm** [bvaisvil/zenith#9]
My system has an NVMe system disk and a ZFS pool of 3 SSDs that is mounted as `/home`. Only the partitions from the NVMe drive appear:

```
┌Disks───────────────────────┐
│9.52%(!): /                 │
│84.71%: /boot               │
│                            │
│                            │
│                            │
│                            │
│                            │
│                            │
└────────────────────────────┘
```

Other tools detect it, for example `ytop`:

```
┌ Disk Usage ────────────────────────────┐
│Partiti Mount   Used  Free     R/s      │
│nvme0n1 /boot    15%  254 MiB  0 bytes  │
│nvme0n1 /        90%  43.3 GiB 2 bytes  │
│zhome   /home    66%  151 GiB  0 bytes  │
│                                        │
│                                        │
│                                        │
│                                        │
│                                        │
│                                        │
│                                        │
└────────────────────────────────────────┘
```

**Comment by bvaisvil** [comment]
Adding zfs pool support definitely on my todo list.

**Comment by bvaisvil** [comment]
Yeah, actually this was a bug, turns out the library wasn't returning all the mounts. Thanks for reporting!

**Comment by wezm** [comment]
Awesome, looks like a huge improvement!

**Comment by bvaisvil** [comment]
This should be fixed in the latest release (0.7.7), or at least my ZFS pool shows up. Feel free to re-open if you have further issues.

**Comment by wezm** [comment]
Great my ZFS pool is appearing and memory use is much improved for me in 0.7.7.
